### ISSUE TITLE
Localize Meshroom

### ISSUE THREAD

**Issue by natowi** [alicevision/meshroom#540]
It should not be too difficult to localize Meshroom.
- [ ] GUI translation [#530]
- [ ] make code translatable and extract strings from source code [[example](https://inventwithpython.com/blog/2014/12/20/translate-your-python-3-program-with-the-gettext-module/)]
- [ ] choose localization platform (transifex, weblate,...)

@jonbragado (Spanish)

---
- [ ] full UTF-8 native support
- [ ] translate documentation (once it is completed)

**Comment by skinkie** [comment]
@skinkie and @baasje85 for Dutch.

**Comment by jonbragado** [comment]
I am not a programmer myself to know enough on how to tweak the source code to make it translatable, but if someone very kindly can help on that and include the qt designer files for the interface windows we can get starting the translation process.

Many thanks,

**Comment by natowi** [comment]
@jonbragado Do you want to use qt linguist? What are the benefits? I would have used python's gettext function as described [here](https://inventwithpython.com/blog/2014/12/20/translate-your-python-3-program-with-the-gettext-module/) and uploaded the strings to [transifex](https://docs.transifex.com/projects/open-source-project) (free for open source projects).

**Comment by jonbragado** [comment]
Yes actually thats the only app i use for translations. QTLinguist is perfect.
Thanks.

**Comment by natowi** [comment]
I will look into this next weekend.
